src/nunit.testmodel/Model/TestModel.cs

```diff
@@ -28,12 +28,16 @@
 
 namespace NUnit.Gui.Model
 {
+    using System.Diagnostics;
```

Comment:
usually we put non-nunit using statements at top of file after header. The ones inside the namespace would be for NUnit usings.